### PR TITLE
Adjusting manifest files for reqmgr2ms

### PR DESCRIPTION
**Pull request by muhammadimranfarooqi**

### CODE REVIEW

**Comment by muhammadimranfarooqi** [comment]
@amaltaro 

Alan, this is new PR which is made according to the changes you suggested in PR [204](https://github.com/dmwm/CMSKubernetes/pull/204)

File: kubernetes/cmsweb/services/reqmgr2ms-output.yaml [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-    app: reqmgr2ms-o
+    app: ms-output
   ports:
-    - port: 8248
-      targetPort: 8248
-      name: reqmgr2ms-o
+    - port: 8245
+      targetPort: 8245
+      name: ms-output
     - port: 18248
```
Comment (amaltaro) [inline kubernetes/cmsweb/services/reqmgr2ms-output.yaml:13]: don't you want to update the port used by prometheus to collect service metrics as well?

File: kubernetes/cmsweb/services/reqmgr2ms-transferor.yaml [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
-    app: reqmgr2ms-t
+    app: ms-transferor
   ports:
-    - port: 8248
-      targetPort: 8248
-      name: reqmgr2ms-t
+    - port: 8247
+      targetPort: 8247
+      name: ms-transferor
     - port: 18248
```
Comment (amaltaro) [inline kubernetes/cmsweb/services/reqmgr2ms-transferor.yaml:13]: same comment here. Shall we update the monitoring port as well?

Comment (muhammadimranfarooqi) [reply kubernetes/cmsweb/services/reqmgr2ms-transferor.yaml:13]: @vkuznet 

Valentin, could you please comment on port number for monitoring.

Comment (vkuznet) [reply kubernetes/cmsweb/services/reqmgr2ms-transferor.yaml:13]: We applied common convention to assign monitoring ports as 10000+port of the service. Therefore, if trasferor port is 8247 the monitoring port would be 18247, etc.

But please keep in mind that in order for Prometheus to start using new service metrics we also need to make changes to Prometheus configuration, see https://github.com/dmwm/CMSKubernetes/blob/master/kubernetes/cmsweb/monitoring/prometheus.yaml

So for all services we have appropriate section there, e.g. for DAS we have
```
      - job_name: 'das-service'
        static_configs:
            - targets: ['das.das.svc.cluster.local:18217']
```
As you can guess you need to create a section for each individual service using <srv>-service as a name, and put in targets list the service entry in a form
```
<servicename>.<servicenamespace>.svc.cluster.local:<port>
```
Therefore, I suggest that you adjust PR to add appropriate section for discussed services.

**Comment by muhammadimranfarooqi** [comment]
@vkuznet 

I have adjusted this PR according to your suggestions. Please take a look. Thanks

**Comment by amaltaro** [comment]
reqmgr2ms.yaml is no longer needed...

**Pull request merged**